index.html

```diff
@@ -0,0 +1,611 @@
+            <h1>Best Way To Present Your Beautiful Mobile App</h1>
+            <p>AppBox is full of awesome features. Its designed for<br> <strong>App Landing and App Store site</strong></p>
+            <figure>
+              <video src="#" width="150" height="85" poster="/img/video-placeholder.jpg"></video>
+              <figcaption>Watch the Intro Video</figcaption>
+            </figure>
+          </div>
+          <div class="hero-image-wrapper">
+            <img src="/img/iphones.png" alt="Image of cool iPhones" title="Cool iPhones">
+          </div>
+        </div>
+        <aside data-aside="get-free" class="download-center">
+          <h2>Get it for free</h2>
+          <p>Choose your native platform and download the app <em>free!</em></p>
+          <ul>
+            <li><a class="btn-large" target="_blank" href="https://play.google.com/store/apps?hl=en"><span class="svg-google"></span>Google Play</a></li>
+            <li><a class="btn-large" target="_blank" href="https://itunes.apple.com/us/genre/ios/id36?mt=8&cid=wwa-us-kwg-features"><span class="icon-apple2"></span>iOS App Store</a></li>
+          </ul>
+        </aside>
+      </section>
+      <section data-section="amazing-features" id="features">
+        <h1 class="header-underline">Some Amazing Features</h1>
+        <ul class="l-flex-container">
+          <li class="flag">
+            <span class="icon-computer"></span>
+            <h2>Responsive design</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-lab"></span>
+            <h2>Easy to customize</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-airplane"></span>
+            <h2>Perfect showcase</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-cog"></span>
+            <h2>App store support</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+        </ul>
+      </section>
+      <section data-section="conference-support">
+        <div class="conference-text-wrapper">
+          <p class="btn-small">New</p>
+          <h1>Live conference support available</h1>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes.</p>
+          <aside>
+            <h2>Free Download On</h2>
+            <div>
+              <a target="_blank" href="https://play.google.com/store/apps?hl=en">
+                <span class="icon-android"></span><br>
+                <p>Android</p>
+              </a>
```

Comment:
There's way more markup than necessary here. The icon should be on the anchor—no need for the `<span>`. Don't use `<br>` tags simply increase line gaps—which this doesn't even accomplish.

This would have been sufficient, along with a little more CSS:

```
<a target="_blank" href="https://play.google.com/store/apps?hl=en" class="icon-android">Android</a>
```

The containing `<div>` is also likely superfluous. A `<ul>` or even a single `<p>` containing both links would have been more appropriate.